demo/chat/package.json

```diff
@@ -4,7 +4,9 @@
   "description": "Demo app for for Amazon Chime Chat SDK",
   "scripts": {
     "deps": "cd ../.. && npm run build && yalc publish",
+    "deps:release": "cd ../.. && npm run build ",
     "start": "npm run deps && yalc add amazon-chime-sdk-component-library-react && npm install && npm run start:fast",
+    "start:release": "npm run deps && yalc add amazon-chime-sdk-component-library-react && npm install && npm run start:fast",
```

Comment:
Remove `yalc` call from here, like is the demo/meeting/package.json?

Reply:
Oops, yes this shouldnt use yalc. Thanks